docs/source/openstack/volume-migration-using-s3.md

```diff
@@ -0,0 +1,190 @@
+| endpoints | moc-kzn                                               |
+|           |   admin: https://kzn-swift.massopen.cloud/swift/v1    |
+|           | moc-kzn                                               |
+|           |   public: https://kzn-swift.massopen.cloud/swift/v1   |
+|           | moc-kzn                                               |
+|           |   internal: https://kzn-swift.massopen.cloud/swift/v1 |
+|           |                                                       |
++-----------+-------------------------------------------------------+
+```
+
+And from the NERC:
+
+```
+$ openstack --os-cloud nerc catalog show object-store -c endpoints
++-----------+----------------------------------------------------------------------------------------+
+| Field     | Value                                                                                  |
++-----------+----------------------------------------------------------------------------------------+
+| endpoints | regionOne                                                                              |
+|           |   admin: http://10.255.2.253:8080                                                      |
+|           | regionOne                                                                              |
+|           |   internal: http://10.255.2.253:8080/v1/AUTH_01234567890123456789012345689012          |
+|           | regionOne                                                                              |
+|           |   public: https://stack.nerc.mghpcc.org:13808/v1/AUTH_01234567890123456789012345689012 |
+|           |                                                                                        |
++-----------+----------------------------------------------------------------------------------------+
+```
+
+## Configure minio client
+
+For each environment, create a MinIO alias configuration using the base URL of the "public" interface of the object-store service and the access key and secret your generated earlier:
+
+```
+$ mc alias set kaizen https://kzn-swift.massopen.cloud 1234567890ABCDEF1234567890ABCDEF 1234567890ABCDEF1234567890ABCDEF
+$ mc alias set nerc https://stack.nerc.mghpcc.org:13808 FEDCBA0987654321FEDCBA0987654321 FEDCBA0987654321FEDCBA0987654321
+```
+
+## Create volume backup in source environment
+
+Create a volume backup in kaizen:
+
+```
+$ openstack --os-cloud kaizen volume backup create --container <bucket_name> <volume id or name>
+```
+
+In the above command line, `<bucket_name>` is the name of the object storage container that will hold your backup. This must be unique across all users, so using your username as a prefix is probably a good idea (e.g., `--container larsks-backups`).
+
+This will create a set of objects in the named bucket. You can use the MinIO client to inspect it:
+
+```
+$ mcli ls kaizen
+[2022-03-17 13:48:47 EDT]     0B larsks-backups/
+$ mcli ls kaizen/larsks-backups/
```

Comment:
mcli gives same error as above but when I try to run it as just mc
```
~/Downloads/mc ls kaizen/jweyl-backups
mc.exe: <ERROR> Unable to list folder. Bucket `jweyl-backups` does not exist.
```
even though the openstack command to create the bucket looks like it worked.
```
openstack --os-cloud kaizen volume backup create --container jweyl-backups 59789d82-37ee-4492-9a51-5d4b47864da4
+-------+--------------------------------------+
| Field | Value                                |
+-------+--------------------------------------+
| id    | c01c881a-77c0-44be-a78f-8a2ad2cdd14a |
| name  | None                                 |
+-------+--------------------------------------+
```

Reply:
and the mc alias seemed to work.
```
~/Downloads/mc alias set kaizen https://kzn-swift.massopen.cloud <access> <secret> 
Added `kaizen` successfully.
```

Reply:
I attempted in ubuntu in kaizen and it returns nothing instead of a volume
```
ubuntu@migration-test:~/.config/openstack$ ./mc ls kaizen/jweyl-backups/                                                             
ubuntu@migration-test:~/.config/openstack$ 
```

Reply:
could the problem be that the backup is in Status error?
```
openstack --os-cloud kaizen volume backup list
+--------------------------------------+------+-------------+--------+------+
| ID                                   | Name | Description | Status | Size |
+--------------------------------------+------+-------------+--------+------+
| 1beb75a6-4ef4-4113-b8ca-5dde5720dde8 | None | None        | error  |    3 |
+--------------------------------------+------+-------------+--------+------+
```

Reply:
I moved mc to `/usr/bin/` and now I can run just mc, still can't run mcli and it still does not see anything when I use `mc ls kaizen/jweyl-backups/`